conda/meta.yaml.official

```diff
@@ -1,24 +1,26 @@
 about:
-  home: http://github.com/UV-CDAT/vcdat
+  home: http://github.com/CDAT/vcdat
 package:
   name: vcdat
-  version: "0.7.0"
+  version: "1.0"
 requirements:
   build:
-  - python
+  - python=2
   - setuptools
   - nodejs
   run:
-  - python
+  - python=2
+  - nodejs
   - flask
-  - vcs 2.12
-  - cdms2 2.12
-  - cdutil 2.12
-  - genutil 2.12
+  - vcs
```

Comment:
hum... We should probably tag vcs 8.0 and cdms 3.0.1 or whatever the latest stable versions are.

Reply:
I updated the tags to the latest version I saw in anaconda. Will have to test it out again tomorrow.